packages/js-sdk/tests/sandbox/files/watch.test.ts

```diff
@@ -31,6 +31,79 @@ sandboxTest('watch directory changes', async ({ sandbox }) => {
   await handle.stop()
 })
 
+sandboxTest('watch recursive directory changes', async ({ sandbox }) => {
+  const dirname = 'test_recursive_watch_dir'
+  const nestedDirname = 'test_nested_watch_dir'
+  const filename = 'test_watch.txt'
+  const content = 'This file will be watched.'
+  const newContent = 'This file has been modified.'
+
+  await sandbox.files.makeDir(`${dirname}/${nestedDirname}`)
+  onTestFinished(() => sandbox.files.remove(dirname))
```

Comment:
Technically correct, but not really necessary, the sandbox is killed after the test ends. This will only add one call to sandbox, but you could do this for local testing. Locally the same sandbox is reused
```suggestion
  if (isDebug) {
    onTestFinished(() => sandbox.files.remove(dirname))
  }
```

Reply:
👍